src/CommentLimit.php

```diff
@@ -14,6 +15,8 @@
  */
 class CommentLimit {
 
+  use StringTranslationTrait;
```

Comment:
Because you added $this->t() calls. Those are only available when you use the StringTranslationTrait.